.github/workflows/windows_release.yml

```diff
@@ -23,11 +23,15 @@ jobs:
     - name: Configure
       shell: bash
       run: |
-          cmake . -Bbuild -G'Visual Studio 16 2019' -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE='C:/projects/vcpkg/scripts/buildsystems/vcpkg.cmake' -DHPX_WITH_PSEUDO_DEPENDENCIES=OFF -DHPX_WITH_EXAMPLES=ON -DHPX_WITH_TESTS=ON -DHPX_WITH_DEPRECATION_WARNINGS=OFF -DHPX_WITH_TESTS_MAX_THREADS_PER_LOCALITY=2
+          cmake . -Bbuild -G'Visual Studio 16 2019' -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE='C:/projects/vcpkg/scripts/buildsystems/vcpkg.cmake' -DHPX_WITH_PSEUDO_DEPENDENCIES=OFF -DHPX_WITH_EXAMPLES=ON -DHPX_WITH_TESTS=ON -DHPX_WITH_TESTS_UNIT=ON -DHPX_WITH_DEPRECATION_WARNINGS=OFF -DHPX_WITH_TESTS_MAX_THREADS_PER_LOCALITY=2
     - name: Build
       shell: bash
       run: |
-          cmake --build build --config Release --target ALL_BUILD -- -maxcpucount -verbosity:minimal -nologo
+          cmake --build build --config Release \
+          --target ALL_BUILD \
+            cmake_build_dir_targets_test cmake_install_dir_targets_test \
+            cmake_build_dir_macros_test cmake_install_dir_macros_test \
```

Comment:
I think these might have to be `cmake_build_dir_targets_test.make_compile` because of the lack of pseudo-dependencies on Windows. @auriane would you mind trying that out? If that doesn't work maybe @hkaiser could check what the targets are called on Windows?

Reply:
will try thanks!